gui/default/syncthing/settings/settingsModalView.html

```diff
@@ -183,7 +183,7 @@
                       <label>
                         <span translate="settings.webauthn.credentials.columnHeading.require2fa">2FA</span>
                         <a href="{{docsURL('users/webauthn') + '#webauthn-require2fa'}}">
-                          <span class="fa fa-question-circle"/>
+                          <span class="fa fa-question-circle" title="Open help: 2FA setting for WebAuthn credentials" />
```

Comment:
@emlun My guess is this will need to be something like:
```
<span class="fa fa-question-circle" tooltip data-original-title="Open help: 2FA setting for WebAuthn credentials"/>
```
but I am not able to test that right now.

Reply:
Yeah, I had a proper tooltip about this at first, but I removed it in favour of just the link to the docs. The tooltip text would had to be really long in order to successfully explain anything, so I figure it's better to just link out to where a proper explanation can fit more comfortably. The `title` attribute was intended mostly for accessibility - for screen readers and things like that.